src/replica_server/replication_lib/mutation.cpp

```diff
@@ -285,7 +286,18 @@ mutation_queue::mutation_queue(gpid gpid, int max_concurrent_op /*= 2*/, bool ba
 
 mutation_ptr mutation_queue::add_work(task_code code, dsn_message_t request, replica* r)
 {
-    // batch and add to work queue
+    task_spec* spec = task_spec::get(code);
+
+    // if not allow write batch, switch work queue
+    if (_pending_mutation && !spec->rpc_request_is_write_allow_batch)
```

Comment:
but it is task-code level option. If I put it in replication-specific option, maybe as:
```
[replication]
rpc_request_disable_batch_write = RPC_RRDB_RRDB_MULTI_PUT,RPC_RRDB_RRDB_INCR
```
or do you have better suggestion?